tests/models/test_taskinstance.py

```diff
@@ -103,6 +103,15 @@ def test_pool():
         session.rollback()
 
 
+@pytest.fixture
+def task_reschedules_for_ti():
+    def wrapper(ti):
+        with create_session() as session:
+            return session.scalars(TaskReschedule.stmt_for_task_instance(ti=ti, descending=False)).all()
+
+    return wrapper
```

Comment:
```suggestion
def task_reschedules_for_ti(session):
    def wrapper(ti):
        return session.scalars(TaskReschedule.stmt_for_task_instance(ti=ti, descending=False)).all()

    return wrapper
```

(I think this should work)

Reply:
I would be better try to avoid to keep session longer than it required